### ISSUE TITLE
Local File Server - Changing root folder?

### ISSUE THREAD

**Issue by NeloBlivion** [Inve1951/BetterDiscordStuff#59]
Is it possible to change the root folder of localfileserver? It's currently set to pictures (C:\Users\Username\Pictures\) but, ironically, all the images i'd like to use are in folders elsewhere.

**Comment by Inve1951** [comment]
Yes, that's merely the default location. You can change the path in the plugin's settings.
![image](https://user-images.githubusercontent.com/18194808/59149854-36d21b00-8a1b-11e9-8e26-8a154c90ec4f.png)

**Comment by Inve1951** [comment]
Nvm, it suffers from the same electron bug as DirectDownload did. I'll fix (i.e. work around) it soon™. In the meanwhile you can change it by editing the `localFileServer.config.json` file. Should look like this:
```json
{
    "settings": {
        "folder": "C:\\Where\\Ever\\You\\Want",
        "port": 443
    }
}
```

**Comment by NeloBlivion** [comment]
Thanks for the help! Editing the config and restarting discord worked great.
